system/ee/ExpressionEngine/Addons/rte/ft.rte.php

```diff
@@ -214,6 +221,7 @@ public function display_field($data)
      */
     public function grid_display_field($data)
     {
+        // @TODO: Repeat this code
         RteHelper::includeFieldResources();
         $configHandle = RteHelper::insertConfigJsById(!empty($this->settings['toolset_id']) ? $this->settings['toolset_id'] : null);
```

Comment:
@dougblackjr does this need to be updated to support Redactor as well?

Reply:
@intoeetive No, with the changes to the helper, it should cover any providers.